### DISCUSSION TITLE
Is it possible to search through windows settings in flow launcher?

### DISCUSSION THREAD

**Discussion by rsmyst** [Flow-Launcher/Flow.Launcher#846] [Q&A] [answered]
I used to use powertoys run so i'm very used to searching for settings through this thing. 
Does flow launcher have a similar plugin? i've seen that it supports cpl but modern windows 10/11 settings support is needed. Last year when I asked about the same, someone told it is on the way... 1 year on the works??

**Comment by jjw24** [comment, marked as answer]
Hey how's it going. it is coming with release 1.9.0, which shouldnt be too long now.
https://github.com/Flow-Launcher/Flow.Launcher/pull/741

**Reply by rsmyst** [reply]
Can't wait. I'm currently using powertoys run and am finding it rather slow to use fully.
Fluent search is what I was originally using and it tended to lag when other apps were in background. 
Flow launcher seems very responsive but it was missing the settings search features. Can't wait!

**Reply by taooceros** [reply]
Which features are you missing now?

**Reply by rsmyst** [reply]
I was only finding the windows settings search to be missing. I couldn't find anything else amiss. Perhaps an option to resize flow launchers bar would be nice. i'd like it to be slightly bigger. The Bar also somehow feels less modern than FS and powertoys run. Maybe because of the font styling size. Or due to flow having squared corners.
Other than that, nothing else! the app was very responsive every time i tried it unlike the others which is a nice plus.

**Reply by jjw24** [reply]
well, you are in luck because resize, round corners are all coming 1.9.0
why not get it early too?
https://github.com/Flow-Launcher/Flow.Launcher/discussions/640

**Reply by rsmyst** [reply]
> well, you are in luck because resize, round corners are all coming 1.9.0 why not get it early too? #640

oh my lord. didn't think i'd hit a jackpot like this. i'll update you any issues i face on 1.9.0 on that thread. thanks a lot!

EDIT: 1.9.0 looks exactly like i wanted. feels very smooth now too! good job devs! can't wait for the public release!
